src/main/Extensions/DataProtection.cs

```diff
@@ -70,4 +70,35 @@ public static class DataProtectionExtensions
             return Encoding.UTF8.GetString(clearBytes);
         }
     }
+    /// <summary>
+    /// forces a re-calculation of the protected data according to current machine setting in EncryptConfig when
+    /// writing the json for renewals and options for plugins
+    /// </summary>
+    public class protectedStringConverter : JsonConverter<string>
```

Comment:
Agreed. One other thought I had was to put this in the RenewalService file, along with a json converter that is currently there. I would argue that there is some sense to see this as part of the renewal service. For now, I'll put it there because I'm not really seeing a great logical place for it and am happy if you want to change it to something else if you have a good idea.